apps/testingaccessibility/src/components/pricing.tsx

```diff
@@ -62,18 +62,22 @@ export const Pricing: React.FC<PricingProps> = ({
   )
 
   const availableCoupon = formattedPrice?.availableCoupons?.[0]
-  const appliedCoupon = formattedPrice?.appliedCoupon
+  const appliedMerchantCoupon = formattedPrice?.appliedMerchantCoupon
```

Comment:
I tried to specify the `merchantCoupon` which is a database reference to the actual stripe coupon 

this is super confusing to me in practice because it requires thinking about which type of coupon we are currently using so being explicit in the name helps. the underlying idea is that we could have multiple merchants (ie paypal), but I'm not sure how that would happen in practice with a global coupon

it might actually be a "many to many" where instead of a "merchantCoupon" a `coupon` would have `merchantCoupons` plural and the final applied coupon would be used for the appropriate merchant

that's complicated future stuff and for now the explicit naming helped me